api/v1alpha1/nonadminbackupstoragelocationrequest_types.go

```diff
@@ -84,6 +84,10 @@ type NonAdminBackupStorageLocationRequestStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=nonadminbackupstoragelocationrequests,shortName=nabslrequest
+// +kubebuilder:printcolumn:name="NABSL-Namespace",type="string",JSONPath=".status.nonAdminBackupStorageLocation.namespace"
+// +kubebuilder:printcolumn:name="NABSL-Name",type="string",JSONPath=".status.nonAdminBackupStorageLocation.name"
```

Comment:
would drop NABSL prefix in namespace title. having just namespace I think is enough m(this is a request for NAMESPACE X)

would also drop NAME suffix for NABSL

Reply:
This object is *mostly* viewable by the cluster admin to interact with and the namespace and name is the one that requested the BSL creation, we can drop it, but would this be easy enough to understand without reading documentation ?